doc-github-rules.md

```diff
@@ -1,16 +1,20 @@
 ---
 title: Github basic process
```

Comment:
I'm not in favor of these header blocks. It takes up space,  doesn't add anything useful - the title should be self-explanatory and the author/date is in the commit info, you're just creating maintenance chores. (Admittedly, if you're not online and reading a printed version, the commit info might not be apparent - but how likely is that?).

Reply:
This is metadata that in online documentation helps in the search. 
Just experimenting with it. 
But I do not know if it helps in our case.

Reply:
Removed metadata.